File: Python/Assignment-2/Prog13.py
income = int(input("Enter your income: "))
incomeTax = 0

if 5000 <= income < 6000:
    incomeTax = 0.1 * (income - 5000)
elif 6000 <= income < 15000:
    incomeTax = 100 + 0.2 * (income - 6000)
elif income >= 15000:
    incomeTax = 1900 + 0.3 * (income - 15000)
else:
    # Assign None for invalid incomes (<= 0), 0 for incomes in the range (0, 5000)
    incomeTax = None if income <= 0 else 0

# Print the result or handle invalid inputs
print(incomeTax if incomeTax is not None else "Invalid Input")

File: C/Assignment-2/Prog13.c
#include<stdio.h>

int main(){
    float income = 0.0, incomeTax = 0.0;
    printf("Enter your income : ");
    scanf("%f", &income);
    
    if (income>=5000 && income<6000)
        incomeTax = (income-5000)*0.1;
    else if(income>=6000 && income<15000)
        incomeTax = (100 + (income-6000)*0.2);
    else
        incomeTax = 1900 + (income-15000);
    
    printf("Income Tax : %.2f", incomeTax);
    return 0;
}